bigbone/src/main/kotlin/social/bigbone/api/entity/PreviewCard.kt

```diff
@@ -107,10 +107,18 @@ data class PreviewCard(
     /**
      * Specifies the type of the preview card.
      */
-    enum class CardType(val value: String) {
-        Link("link"),
-        Photo("photo"),
-        Video("video"),
-        Rich("iframe")
+    @Serializable
+    enum class CardType {
+        @SerialName("link")
+        LINK,
+
+        @SerialName("photo")
+        PHOTO,
+
+        @SerialName("video")
+        VIDEO,
+
+        @SerialName("iframe")
+        RICH
```

Comment:
On https://docs.joinmastodon.org/entities/PreviewCard/#type I can see that this should be `rich` instead of `iframe`? 🤔 I can see that it was `iframe` before your changes but that seems to no longer be valid.

```suggestion
        @SerialName("rich")
        RICH
```